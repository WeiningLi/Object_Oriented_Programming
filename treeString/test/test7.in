ra ""
^a0 "a"
^a1 "b"
^a2 "c"
^a3 "348274829374982721730937403"
^a12 "kjhlqkwje;wqjelkjwqfljqw;"
^a7 " wf ef  ewf w wdw d w"
pa
da
cba
pb
q
