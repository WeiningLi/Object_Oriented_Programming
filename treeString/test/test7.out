abc3482 wf ef  ewf w wdw d w74829kjhlqkwje;wqjelkjwqfljqw;374982721730937403
"a" size: 1
 "b" size: 1
  "c" size: 1
   "kjhlqkwje;wqjelkjwqfljqw;" size: 55
    " wf ef  ewf w wdw d w" size: 25
     "3482" size: 4
     "74829" size: 5
    "374982721730937403" size: 18
abc3482 wf ef  ewf w wdw d w74829kjhlqkwje;wqjelkjwqfljqw;374982721730937403
